honeybadger/connection.py

```diff
@@ -28,8 +28,14 @@ def send_request():
         if status != 201:
             logger.error("Received error response [{}] from Honeybadger API.".format(status))
 
+        response_json = json.loads(response.read().decode('utf-8'))
```

Comment:
Wouldn't it be safer if you wrapped this in a try/catch block? We don't want Honeybadger throwing uncaught errors in case of network issues 😄 . Additionally, I don't know about our python client, but for other clients, an uncaught error in this place would create a loop (possibly infinite).